chalice/app.py

```diff
@@ -205,6 +205,21 @@ def serve_swagger_ui():
                                 headers={"Content-Type": "text/html"},
                                 body=swagger_ui_html)
 
+    @app.route("/version")
+    @time_limited(app)
+    def version():
+        data = {
+            'version_info': {
+                'dss_version': os.environ['DSS_VERSION']
```

Comment:
I would move the version to a static variable. That would prevent the application from making a system call every time there is a `GET /version`

Reply:
Also, I would check if the version endpoints should follow a similar JSON schema between DCP components. `dss_version` is a key specific to DSS. Perhaps it would be good to have them all similar?

Reply:
After a conversation with @sampierson, we agreed that some DCP components aren't/can't be described by a single component version. For reference, I followed the format of [Green box's `/version` endpoint](http://pipelines.dev.data.humancellatlas.org/version), using specific subcomponent names. However, I see your point and would propose that we:
- maintain `version_info` key
- use `version` as the component's version key
- use `<subcomponent>_version` as subcomponents' version key
- standardize this format across all `/version` endpoints

Reply:
Ah I see, that is the standard. Go ahead with that, then.

Reply:
Getting an environment variable doesn't involve a syscall, FYI.

Reply:
@ttung [true](https://stackoverflow.com/questions/46623018/does-a-program-make-a-system-call-to-get-the-value-of-an-environment-variable-in). Good catch.